modules/market/src/main/java/com/opengamma/strata/market/curve/CurveCalibrationInfo.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.market.curve;
+
+/**
+ * Information about curve calibration.
+ * <p>
+ * Implementations of this interface provide information that was obtained during curve calibration.
+ */
+public interface CurveCalibrationInfo {
```

Comment:
It smells like a marker interface that might or might not know how to convert itself to another type. But probably doesn't. Despite my distaste for marker interfaces, why not just use one rather than having the `convertTo` method? It would be more conventional.